docs/sql/functions/nested.md

```diff
@@ -161,6 +161,8 @@ SELECT * FROM range(date '1992-01-01', date '1992-03-01', interval '1' month);
 
 The function `list_aggregate` allows the execution of arbitrary existing aggregate functions on the elements of a list. Its first argument is the list (column), its second argument is the aggregate function name, e.g. `min`, `histogram` or `sum`.
 
+`list_aggregate` accepts additional arguments after the aggregate function name. These extra arguments provide customization options to tailor the aggregation process according to specific requirements.
```

Comment:
Maybe tweak this to mention that they're passed to the aggregate function? Instead of vague "specific requirements"